src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -1,4 +1,4 @@
-import urllib
+import urllib.parse
```

Comment:
Further down, `urllib.parse.quote()` is used. For this, I don't get code insights in my IDE with `import urllib`. Only when changing this to `import urllib.parse`, I get intellisense / code hover support